Framework/Reflectometry/test/SaveILLCosmosAsciiTest.h

```diff
@@ -8,16 +8,16 @@
 
 #include "MantidAPI/AnalysisDataService.h"
 #include "MantidAPI/MatrixWorkspace.h"
-#include "MantidDataHandling/SaveILLCosmosAscii.h"
+#include "MantidReflectometry/SaveILLCosmosAscii.h"
```

Comment:
You can remove this algorithm altogether, this is deprecated in favour of SaveReflectometryAscii.

Reply:
This is currently still used by the ISIS Reflectometry GUI. We'll look at removing it in #24429